### ISSUE TITLE
Let'sEcnrypt

### ISSUE THREAD

**Issue by pmoncho99** [Ylianst/MeshCentral#74]
Error -

```
-------- 1/7/2019, 6:39:22 PM ---- 0.2.5-x --------

ERROR: Let's encrypt error:  { Error: Could not locate the bindings file. Tried:
 → /home/user99/mc/node_modules/ursa-optional/build/ursaNative.node
 → /home/user99/mc/node_modules/ursa-optional/build/Debug/ursaNative.node
 → /home/user99/mc/node_modules/ursa-optional/build/Release/ursaNative.node
 → /home/user99/mc/node_modules/ursa-optional/out/Debug/ursaNative.node
 → /home/user99/mc/node_modules/ursa-optional/Debug/ursaNative.node
 → /home/user99/mc/node_modules/ursa-optional/out/Release/ursaNative.node
 → /home/user99/mc/node_modules/ursa-optional/Release/ursaNative.node
 → /home/user99/mc/node_modules/ursa-optional/build/default/ursaNative.node
 → /home/user99/mc/node_modules/ursa-optional/compiled/8.10.0/linux/x64/ursaNative.node
    at bindings (/home/user99/mc/node_modules/bindings/bindings.js:96:9)
    at Object. (/home/user99/mc/node_modules/ursa-optional/lib/ursa.js:18:37)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at module.exports (/home/user99/mc/node_modules/rsa-compat/lib/generate-privkey-ursa.js:14:12)
    at module.exports (/home/user99/mc/node_modules/rsa-compat/lib/generate-privkey.js:20:51)
    at Object.RSA.generateKeypair (/home/user99/mc/node_modules/rsa-compat/lib/rsa.js:37:51)
    at Object.generateKeypairAsync (internal/util.js:230:26)
    at /home/user99/mc/node_modules/greenlock/lib/core.js:93:24
    at 
  tries: 
   [ '/home/user99/mc/node_modules/ursa-optional/build/ursaNative.node',
     '/home/user99/mc/node_modules/ursa-optional/build/Debug/ursaNative.node',
     '/home/user99/mc/node_modules/ursa-optional/build/Release/ursaNative.node',
     '/home/user99/mc/node_modules/ursa-optional/out/Debug/ursaNative.node',
     '/home/user99/mc/node_modules/ursa-optional/Debug/ursaNative.node',
     '/home/user99/mc/node_modules/ursa-optional/out/Release/ursaNative.node',
     '/home/user99/mc/node_modules/ursa-optional/Release/ursaNative.node',
     '/home/user99/mc/node_modules/ursa-optional/build/default/ursaNative.node',
     '/home/user99/mc/node_modules/ursa-optional/compiled/8.10.0/linux/x64/ursaNative.node' ] }
```
Current Config File
```
{
  "settings": {
    "WANonly": true,
    "_Minify": 1,
    "SessionTime": 30,
    "Port": 443,
    "RedirPort": 80,
    "SelfUpdate": true,
    "UserAllowedIP": "externalIPs",
    "_WebRtConfig": { "iceServers": [ { "urls": "stun:stun.services.mozilla.com"
 }, { "urls": "stun:stun.l.google.com:19302" } ] }
  },
  "letsencrypt": {
    "email": "user@mydomain.com",
    "names": "domain1.net",
    "rsaKeySize": 3072,
    "production": false
  },
}

**Comment by Ylianst** [comment]
Thanks. Published MeshCentral v0.2.6-e with fix for this. MeshCentral will try to install GreenLock without optional modules and on top of that, it will automatically try to remove the "ursa-optional" folder. Hopefully, this will solve this issue going forward.

**Comment by aullah360** [comment]
is anybody can instruct me to install let'sEncript for my Meshcentral Server which is hosted on a Windows 10 PC. Also, it has been pointed to a subdomain. 
what is the procedure to get the Let' sEncrypt on my server?

Thanks
